assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -4,22 +4,57 @@ import Table from '@components/Table';
 import ReactMarkdown from 'react-markdown';
 import remarkGfm from 'remark-gfm';
 import LoadingBox from '@components/LoadingBox';
+import Modal from '@components/Modal';
 
-import { getCheckResults, getCheckDescription } from './checksUtils';
+import {
+  getCheckResults,
+  getCheckDescription,
+  getCheckRemediation,
+} from './checksUtils';
 
 import ResultsContainer from './ResultsContainer';
 import { ExecutionIcon } from './ExecutionIcon';
 import CheckResultOutline from './CheckResultOutline';
 import ExecutionHeader from './ExecutionHeader';
 
+const addHostnameToTargets = (targets, hostnames) =>
+  targets?.map((target) => {
+    const { agent_id } = target;
+
+    const { hostname } = hostnames.find(({ id }) => agent_id === id);
+    return {
+      ...target,
+      hostname,
+    };
+  });
+
+function MarkdownContent({ children }) {
```

Comment:
I know that you are simply moving this code from place, but...
Do we need this markdown wrapper?
It doesn't do anything, we could simply use the one inside, don't we?

Reply:
My fault :sweat_smile: just to reduce a bit of boiler... ocd you konw...

However nothing against removing.

Reply:
Both fine by me, but i remove them.